File: Assignments/Assignment_13/p3.c
#include<stdio.h>
void main(){

        int row;
        printf("Enter the no of rows : \n");
        scanf("%d",&row);
        for(int i=1;i<=row;i++){

                for(int j=0;j<i;j++){

                        printf("%c\t",(64+i+j));
                }
                printf("\n\n");
        }
}

File: Flutter/Assignment/Square pattern set-2/p3.dart
import "dart:io";
void main(){
	print("Enter the no. of rows:");
	int? row=int.parse(stdin.readLineSync()!);
	for(int i=1;i<=row;i++){
		int x=1;
		for(int j=1;j<=row;j++){
			if(j==(row-i+1)){
				stdout.write("*\t");
				x++;
			}else{
				stdout.write("${x}\t");
				x++;
			}
		}
		print("");
	}
}
